strawberry/schema/schema_converter.py

```diff
@@ -483,10 +483,11 @@ def _get_result(_source: Any, info: Info, **kwargs):
                 source=_source, info=info, kwargs=kwargs
             )
 
-            if field.base_resolver:
-                return field.base_resolver(*field_args, **field_kwargs)
+            field.default_resolver = self.config.default_resolver  # type: ignore
```

Comment:
Does this need to be set here or could it be set outside of `_get_result`? If I understand this code correctly, when defining this function we already have the config set, and it isn't something that should be changed at runtime

Reply:
yes, I can move that 😊 will do that later 😊